src/vfio_device.rs

```diff
@@ -968,3 +968,69 @@ impl Drop for VfioDevice {
         self.container.put_group(self.group.clone());
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use kvm_ioctls::Kvm;
+    use kvm_bindings::kvm_device_type_KVM_DEV_TYPE_VFIO;
+
+    // Real VFIO device with VFIO_DEVICE_PATH is used to test
+    const VFIO_DEVICE_PATH: &str = "/sys/bus/pci/devices/0000:00:14.0/";
```

Comment:
What type of this PCI device? Is it available on all intel platforms? Is it safe to use it for tests? Should we limit this test to x86 platforms?

Reply:
It's available on local platform can pass the unit test. Since VFIO needs a real PCI device to be performed, and what kind of a device depends on the test environment. We use this local device as an example, any this device-specific test will be ignored, or test case can be run if change the device path.

Reply:
It's available on local platform can pass the unit test. Since VFIO needs a real PCI device to be performed, and what kind of a device depends on the test environment. We use this local device as an example, any this device-specific test will be ignored, or test case can be run if change the device path.

Reply:
we have two unit tests, test_create_vfio_container which only depends on KVM support on the host,  test_vfio_device which require a real PCI device, it is very hard to find a common PCI device to test especially the CI/CD may test on the cloud environment, that’s the reason we ignore this test by default. This test is mainly help developer verify their code locally.

Reply:
> I think the `test_vfio_device` there is useful, but it needs to be documented a little more:
> 
> 1. Specify that this PCI path is used by a test that's ignored by default
> 2. Describe how one could pick a local device for testing
> 3. Describe how this device should unbound from its regular driver and bound back to vfio
> 4. Descrive how the test can then be enabled back

I have updated with more document :)